src/Data/YAML/Event/Internal.hs

```diff
@@ -72,10 +74,18 @@ data Directives = NoDirEndMarker    -- ^ no directives and also no @---@ marker
 data ScalarStyle = Plain
                  | SingleQuoted
                  | DoubleQuoted
-                 | Literal
-                 | Folded
+                 | Literal !Chomp !IndentOfs
+                 | Folded !Chomp !IndentOfs
                  deriving (Eq,Ord,Show)
 
+data Chomp = Strip -- ^ Remove all trailing line breaks.
```

Comment:
`@since 0.2.0` annotations in haddock docstrings would be good; and also add docstrings for all new stuff please :-)

Reply:
Further Documentation for Writer.hs will be done soon